pkg/action/util/util.go

```diff
@@ -318,3 +325,123 @@ func GetPodGrandInfo(kclient *client.Clientset, pod *api.Pod) (string, string, e
 
 	return kind, name, nil
 }
+
+// Updates replica of a controller.
+// Currently, it supports replication controllers, replica sets, and deployments
+func UpdateReplicas(kubeClient *client.Clientset, namespace, contName, contKind string, newReplicas int) error {
+	newValue := int32(newReplicas)
+	switch contKind {
+	case util.KindReplicationController:
+		cont, err := kubeClient.CoreV1().ReplicationControllers(namespace).Get(contName, metav1.GetOptions{})
+		if err != nil {
+			return err
```

Comment:
log the error here, so it will be easier for debug

Reply:
This function is similar to those in [action/executor/hscale_helper.go](https://github.com/turbonomic/kubeturbo/blob/a682e010ee5a4d9c5cf60b3c3e24f38b3a2bb173/pkg/action/executor/hscale_helper.go#L121)

Reply:
fixed